src/CacheInterface.php

```diff
@@ -2,12 +2,22 @@
 
 namespace React\Cache;
 
+use React\Promise\PromiseInterface;
+
 interface CacheInterface
 {
-    // @return React\Promise\PromiseInterface
+    /**
+     * @return PromiseInterface
+     */
     public function get($key);
 
+    /**
+     * @return PromiseInterface
+     */
     public function set($key, $value);
 
+    /**
+     * @return PromiseInterface
```

Comment:
This might not be the best place to discuss this topic. This probably more suited for a meta discussion. For now, i'd propose to simply define the the possible resolution values in the description.

```php
/**
 * @return PromiseInterface A promise which resolves to TRUE if the entry was successfully stored in the cache, FALSE otherwise.
 */
public function set($key, $value);

/**
 * @return PromiseInterface A promise which resolves to TRUE if the cache entry was successfully removed, FALSE otherwise.
 */
public function remove($key);
```